ckanext/scheming/fanstatic/scheming-repeating-subfields.js

```diff
@@ -23,10 +23,17 @@ this.ckan.module('scheming-repeating-subfields', function (jQuery, _) {
 
             $remove.on('click', function(e) {
                 var $groups = $this.find('.scheming-subfield-group'),
-                    $last = $groups.last(),
+                    $curr = $(this).closest('.scheming-subfield-group'),
+                    index = $curr.data('groupIndex'),
+                    field = $curr.data('field'),
                     count = $groups.length;
                 if(count !== 1) {
-                    $last.remove();
+                    document.getElementById("statusText").innerHTML = "Are you sure you want to remove "+field+" "+(index+1)+"?";
```

Comment:
Need to make this translatable, look at the other js files in ckan for an example

Reply:
What is the best way to make it translatable? So far I've used `{% set header = {'en':'Removal Confirmation','fr':'Confirmation de Suppression'} %}` and `{{ h.scheming_language_text(header) }}` for en/fr. Is this general enough?

Reply:
no, just use normal gettext calls `_()` so that we can use existing system-wide translations